### PR TITLE
[dv,flash_ctrl] stress_all test

### PR DESCRIPTION
**Pull request by jdonjdon**
- Add stress_all test as part of V2 requirement.
 - More qualified tests will be added to 'stress_all_vseq'

### CODE REVIEW

**Review by eshapira — APPROVED** [review]
LGTM with one question.

File: hw/ip/flash_ctrl/dv/flash_ctrl_base_sim_cfg.hjson [hunk 1]
```diff
@@ -170,5 +170,12 @@
       name: smoke
       tests: ["flash_ctrl_smoke"]
     }
+    {
+      // For test clean up run subset of tests
+      name: sub
```
Comment (weicaiyang) [inline hw/ip/flash_ctrl/dv/flash_ctrl_base_sim_cfg.hjson:175]: nit: perhaps use a more meaningful name, like all_smoke or baisc_tests

BTW, you can also use regex to run these 2 tests in one shot - `-i *_smoke*`. `-i` supports regex.

**Review by weicaiyang — APPROVED** [review]
LGTM with a few nits

**Comment by weicaiyang** [comment]
sorry for the late review. Since already passes CI, let me merge it and you can address those nits in another PR

**Pull request merged**